File: solutions/onechicken.cpp
#include <bits/stdc++.h>
using namespace std;

int main() {
    ios::sync_with_stdio(false);
    cin.tie(nullptr);

    int n, m;
    cin >> n >> m;
    int diff;

    if (m >= n) {
        diff = m - n;
        if (diff == 1 || diff == 0) {
            cout << "Dr. Chaz will have " << diff << " piece of chicken left over!\n";
        } else {
            cout << "Dr. Chaz will have " << diff << " pieces of chicken left over!\n";
        }
    } else {
        diff = n - m;
        if (diff == 1 || diff == 0) {
            cout << "Dr. Chaz needs " << diff << " more piece of chicken!\n";
        } else {
            cout << "Dr. Chaz needs " << diff << " more pieces of chicken!\n";
        }
    }
}

File: solutions/onechicken.java
/*
https://open.kattis.com/problems/onechicken

Two cases, brute force
*/
import java.util.*;
import java.io.*;
import java.math.*;

public class onechicken{
	public static void main (String[] args) throws IOException {
		BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
		PrintWriter pw = new PrintWriter(new OutputStreamWriter(System.out));

		String[] temp=br.readLine().split(" ");;
		int n=Integer.parseInt(temp[0]);
		int m=Integer.parseInt(temp[1]);
		if (n<m){
			if (m-n == 1){pw.println("Dr. Chaz will have 1 piece of chicken left over!");}
			else {pw.println("Dr. Chaz will have "+ (m-n) +" pieces of chicken left over!");}
		}
		else{
			if (n-m == 1){pw.println("Dr. Chaz needs 1 more piece of chicken!");}
			else{pw.println("Dr. Chaz needs "+ (n-m) +" more pieces of chicken!");}
		}
		pw.flush();
		pw.close();
	}
}
